cogs/trading.py

```diff
@@ -364,8 +364,9 @@ async def confirm(self, ctx):
         if self.bot.trades[ctx.author.id]["executing"]:
             return await ctx.send("The trade is currently loading...")
         
-        if len(self.bot.trades[ctx.author.id]["pokemon"]) <= 0:
-            return await ctx.send("There are no pokemons in this trade!!!")
+        trade = self.bot.trades[user.id]
+        if trade["pokecoins"] <= 0 and trade['redeems'] <= 0 and len(trade['pokemon']) <= 0:
+            return await ctx.send("There are no pokémon/redeems/pokécoins in this trade!")
```

Comment:
Replace message with "This trade is empty! Please run `p!trade cancel`" or something more concise/informative.

Reply:
Ok sure, I'll change that..